### PR TITLE
Create page for dual stack installations

### PR DESCRIPTION
**Pull request by jacob-delgado**
Turn https://istio.io/latest/blog/2023/experimental-dual-stack/ into an installation page

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [ ] Docs
- [X] Installation
- [X] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/setup/additional-setup/dual-stack/index.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+{{< warning >}}
+It is not clear if this works well with Ambient.
+{{< /warning >}}
+
+## Feedback
```
Comment (kfaseela) [inline content/en/docs/setup/additional-setup/dual-stack/index.md:73]: May be we can have some verification examples as well?

File: content/en/docs/setup/additional-setup/dual-stack/index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+title: Install Istio on a Dual-Stack Kubernetes Cluster
+description: Install and use Istio in Dual-Stack mode running on a Dual-Stack Kubernetes cluster.
+weight: 70
+aliases:
+    - /docs/setup/kubernetes/install/dual-stack
+    - /docs/setup/kubernetes/additional-setup/dual-stack
+keywords: [dual-stack]
+owner: istio/wg-networking-maintainers
+test: yes
```
Comment (ericvn) [inline content/en/docs/setup/additional-setup/dual-stack/index.md:10]: We have tests set to `yes` which generated the snips file, but we don't have any tests.

**Comment by linsun** [comment]
cc @ilrudie

File: content/en/docs/setup/additional-setup/dual-stack/index.md
Comment (ilrudie) [inline content/en/docs/setup/additional-setup/dual-stack/index.md:1]: Should we have any details about CNI compatibility? Working, not working, untested/unknown status?

File: content/en/docs/setup/additional-setup/dual-stack/index.md (hunk 1 above)
Comment (ilrudie) [reply content/en/docs/setup/additional-setup/dual-stack/index.md:73]: The associated blog has details about testing TCP which we might point folks to. I'm also working on some httpbin/helloworld type stuff and could contribute details about verifying using one of those or perhaps works on a follow up blog post.

File: content/en/docs/setup/additional-setup/dual-stack/index.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+* Istio 1.17 or later.
+
+## Installation
+
+The installation guide requires changes to both istiod and the injected sidecars. When
+installing Istio you will need to modify your `IstioOperator` or Helm values overrides
+files with the following when you are installing Istio.
+
+{{< tip >}}
+Currently Istio only supports dual stack clusters on initial installation. While it is possible
```
Comment (windsonsea) [inline content/en/docs/setup/additional-setup/dual-stack/index.md:27]:
```suggestion
Currently Istio only supports dual-stack clusters on initial installation. While it is possible
```
Keep consistent in a single sentence with the term `dual-stack`

**Review by craigbox — COMMENTED** [review]
@jacob-delgado bringing this back to top of mind; just needs a test (or tests removed) and a decision on extra examples.

**Comment by kfaseela** [comment]
@jacob-delgado we would need something similar for dual-stack promotion to alpha, and I plan to take this forward, with additional docs tests.. hope you are okay with that

**Pull request closed without merging**